### ISSUE TITLE
CS0433 compiler error: ValueTask<TResult> is defined in both System.Runtime and System.Threading.Tasks.Extensions

### ISSUE THREAD

**Issue by natemcmaster** [dotnet/corefx#25898]
**Repro steps:**

1. git clone https://github.com/aspnet/Common.git
2. Checkout this commit: https://github.com/aspnet/Common/commit/2c4a50456614ea3b0a21aef00adf13581299cafd
3. build.cmd

**Result**:
```
C:\dev\aspnet\Universe\modules\Common\test\Microsoft.Extensions.Internal.Test\ObjectMethodExecutorTest.cs(500,20): 
error CS0433: The type 'ValueTask<TResult>' exists in both 'System.Threading.Tasks.Extensions, Version=4.1.2.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' and 'System.Runtime, Version=4.2.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' [C:\dev\aspnet\Universe\modules\Common\test\Microsoft.Extensions.Internal.Test\Microsoft.Extensions.Internal.Test.csproj]
C:\dev\aspnet\Universe\modules\Common\test\Microsoft.Extensions.Internal.Test\ObjectMethodExecutorTest.cs(505,20): 
error CS0433: The type 'ValueTask<TResult>' exists in both 'System.Threading.Tasks.Extensions, Version=4.1.2.0, Culture=neutral, PublicKeyToken=cc7b13ffcd2ddd51' and 'System.Runtime, Version=4.2.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' [C:\dev\aspnet\Universe\modules\Common\test\Microsoft.Extensions.Internal.Test\Microsoft.Extensions.Internal.Test.csproj]
```

**Details**:

Using 
System.Threading.Tasks.Extensions/4.5.0-preview1-26008-09
Microsoft.NETCore.App/2.1.0-preview1-26009-01

**Log**
[msbuild.binlog.zip](https://github.com/dotnet/corefx/files/1557047/msbuild.binlog.zip)

cc @dagood @weshaggard

**Comment by weshaggard** [comment]
This was moved to System.Runtime and inbox as part of change https://github.com/dotnet/corefx/pull/22559. However we didn't correctly update the configuration.props (https://github.com/dotnet/corefx/blob/master/src/System.Threading.Tasks.Extensions/src/Configurations.props and https://github.com/dotnet/corefx/blob/master/src/System.Threading.Tasks.Extensions/ref/Configurations.props)  to put the netcoreapp configurations in the individual library package to indicate that they are type-forwarded now. We should use this issue to track fixing that. 

@natemcmaster To unblock yourself you can remove the package reference to System.Threading.Tasks.Extensions as it is now part of NETCore.App.

**Comment by natemcmaster** [comment]
Thanks. I'll look into that workaround

**Comment by weshaggard** [comment]
Fixed with #25932
